### ISSUE TITLE
AttributeError: 'BlueskyEventStream' object has no attribute '_user_parameters'

### ISSUE THREAD

**Issue by prjemian** [bluesky/hklpy#213]
```py
In [9]: list_orientation_runs(databroker.catalog["class_2021_03"], "energy")
AttributeError: 'BlueskyEventStream' object has no attribute '_user_parameters'
```

**Comment by prjemian** [comment]
```py
In [16]: databroker.__version__
Out[16]: '1.2.4'
```

**Comment by prjemian** [comment]
Resolved by both #238 and #240.
